README.md

```diff
@@ -19,15 +19,34 @@ pip install log-enricher
 configuration
 -------------
 
-The log-enricher is configured with a simple dictionary:
-```
+Log-enricher runs Enrichers, and is configured with a simple dictionary:
+
+```python
+from log_enricher import initialize_logging
+
+from log_enricher.enrichers import Enricher
+
+class RequestIDEnricher(Enricher):
+    def __call__(self) -> Dict[str, Any]:
+        return {"request_id": get_request_id()}
+
 config = {
     "handlers": "plain"/"structured",
     "log_level": "DEBUG",
     "app_version": "xyz",
     "release_stage": "staging",
     "loggers": ["py", "mylogger"]
 }
+
+initialize_logging(config, enrichers=[RequestIDEnricher()])
 ```
+
 Logs will be output in a plain, console-friendly format if `handlers` is 
-`"plain"`, or in a structured JSON format if it is `"structured"`.
\ No newline at end of file
+`"plain"`, or in a structured JSON format if it is `"structured"`.
+
+`initialize_logging()` will talk to and configure the `logging` subsystem. Once that has been called, `logging`
+will use the new configuration.
+
+# TODO: HOW DO ENRICHERS WORK?
+
+# TODO: WHAT OPTIONS ARE ALLOWED IN THE CONFIG DICT?
```

Comment:
@arni-inaba, so this is a set of things that would be allowed in the config:
```"handlers": "plain"/"structured",
     "log_level": "DEBUG",		     "log_level": "DEBUG",
     "app_version": "xyz",		     "app_version": "xyz",
     "release_stage": "staging",		     "release_stage": "staging",
     "loggers": ["py", "mylogger"]		     "loggers": ["py", "mylogger"]```

What values are allowed and what mechanisms do they control?

Reply:
Ah, it basically prepares things for `logging.dictConfig`, right? https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig

Reply:
Précis!
Handlers, loggers and log_level are used that way, app_version and release_stage are used for enrichers.